tests/acceptance/helpers/webdavHelper.js

```diff
@@ -13,7 +14,7 @@ const occHelper = require('../helpers/occHelper')
 exports.createDavPath = function (userId, element) {
   return client.globals.backend_url +
     '/remote.php/dav/files/' + userId + '/' + encodeURIComponent(element)
-    .replace('%2F', '/')
+    .replace(/%2F/g, '/')
```

Comment:
`replace()` only replaces a single occurrence. If we use this pattern with `/g` flag, then, it will replace all occurrences.